include/odp/api/spec/ipsec.h

```diff
@@ -191,6 +233,42 @@ typedef struct odp_ipsec_outbound_config_t {
 
 } odp_ipsec_outbound_config_t;
 
+/**
+ * IPSEC DEBUG capability
+ */
+typedef struct odp_ipsec_debug_capability_t {
+	/** Support for updating internal parameters of the SA for debug */
+	odp_support_t sa_update;
+
+	/** Parameters supported for sa_update */
+	struct {
+		/** Inbound specific internal parameters */
+		union {
+			/** Mapping for individual bits */
+			struct {
+				/** Antireplay window top */
+				uint32_t antireplay_window_top  : 1;
```

Comment:
I am not sure the capabilities should be defined as bit fields as it does not really save memory or cycles. Perhaps these could be just odp_bool_t (or odp_support_t?). Maybe Petri can comment?

Reply:
After discussion in call, boolean makes most sense.

Reply:
Changed antireplay_window_top and seq_num from bit fields to odp_bool_t in v3.